pkgs/applications/office/pympress/default.nix

```diff
@@ -23,16 +26,22 @@ python3Packages.buildPythonApplication rec {
   buildInputs = [
     gtk3
     gobject-introspection
-    libcanberra-gtk3
     poppler_gi
-  ];
+  ] ++ lib.optional withGstreamer libcanberra-gtk3;
 
   propagatedBuildInputs = with python3Packages; [
     pycairo
     pygobject3
-    python-vlc
+    setuptools
     watchdog
-  ];
+  ] ++ lib.optional withVLC python-vlc;
+
+  patchPhase = ''
+    # Current setuptools version 57.2.0.post0 breaks int(n) in line
+    # https://github.com/Cimbali/pympress/blob/v1.6.2/setup.py#L369 (added in 1.6.x)
+    substituteInPlace setup.py \
+      --replace "setuptools.__version__.split('.')" "setuptools.__version__.split('.')[:2]"
```

Comment:
Can you fetchpatch this commit? https://github.com/Cimbali/pympress/pull/208/commits/9ef5d1457fe5c576707041a3770c8042a985481d
I reported it upstream.

Reply:
Done in latest force-push.